test/integration/singlecluster/scheduler/fairsharing/fair_sharing_test.go

```diff
@@ -293,108 +282,46 @@ var _ = ginkgo.Describe("Scheduler", func() {
 	})
 
 	ginkgo.When("using hierarchical cohorts", func() {
-		var (
-			cohortFirstLeft   *kueuealpha.Cohort
-			cohortFirstRight  *kueuealpha.Cohort
-			cohortSecondLeft  *kueuealpha.Cohort
-			cohortSecondRight *kueuealpha.Cohort
-			cohortBank        *kueuealpha.Cohort
-			cqSecondLeft      *kueue.ClusterQueue
-			cqSecondRight     *kueue.ClusterQueue
-			lqSecondLeft      *kueue.LocalQueue
-			lqSecondRight     *kueue.LocalQueue
-		)
-
-		ginkgo.BeforeEach(func() {
+		ginkgo.It("admits workloads respecting fair share", func() {
 			//         root
 			//        /    \
 			//       fl     fr
 			//     /   \     \
 			//    sl   sr    bank
 			//    /     \
 			//   cqL    cqR
-
-			cohortFirstLeft = testing.MakeCohort("first-left").Parent("root").Obj()
-			gomega.Expect(k8sClient.Create(ctx, cohortFirstLeft)).To(gomega.Succeed())
-
-			cohortFirstRight = testing.MakeCohort("first-right").Parent("root").Obj()
-			gomega.Expect(k8sClient.Create(ctx, cohortFirstRight)).To(gomega.Succeed())
-
-			cohortSecondLeft = testing.MakeCohort("second-left").Parent(kueue.CohortReference(cohortFirstLeft.Name)).Obj()
-			gomega.Expect(k8sClient.Create(ctx, cohortSecondLeft)).To(gomega.Succeed())
-
-			cohortSecondRight = testing.MakeCohort("second-right").Parent(kueue.CohortReference(cohortFirstLeft.Name)).Obj()
-			gomega.Expect(k8sClient.Create(ctx, cohortSecondRight)).To(gomega.Succeed())
-
-			cohortBank = testing.MakeCohort("bank").Parent(kueue.CohortReference(cohortFirstRight.Name)).
+			cohortFirstLeft := createCohort(testing.MakeCohort("first-left").Parent("root").Obj())
+			cohortFirstRight := createCohort(testing.MakeCohort("first-right").Parent("root").Obj())
+			cohortSecondLeft := createCohort(testing.MakeCohort("second-left").Parent("first-left").Obj())
+			cohortSecondRight := createCohort(testing.MakeCohort("second-right").Parent("first-left").Obj())
+			cohortBank := createCohort(testing.MakeCohort("bank").Parent("first-right").
 				ResourceGroup(
 					testing.MakeFlavorQuotas(defaultFlavor.Name).Resource(corev1.ResourceCPU, "10").FlavorQuotas,
-				).Obj()
-			gomega.Expect(k8sClient.Create(ctx, cohortBank)).To(gomega.Succeed())
+				).Obj())
 
-			cqSecondLeft = testing.MakeClusterQueue("second-left").
-				Cohort(kueue.CohortReference(cohortSecondLeft.Name)).
+			cqSecondLeft := createQueue(testing.MakeClusterQueue("second-left").
+				Cohort("second-left").
```

Comment:
nit: I would probably use the names from objects like before `cohortSecondLeft.Name`